numpy/core/src/umath/ufunc_object.c

```diff
@@ -555,8 +555,9 @@ ufunc_get_name_cstr(PyUFuncObject *ufunc) {
  * Parses the positional and keyword arguments for a generic ufunc call.
  *
  * Note that if an error is returned, the caller must free the
- * non-zero references in out_op.  This
- * function does not do its own clean-up.
+ * non-zero references in out_op.  This function does not do its own clean-up.
```

Comment:
It'd be nice to change it so it does this too, but I suppose there's no need for that to be part of _this_ PR

Reply:
Yes, I thought about that too, and am happy to do a follow-up PR for that; there is no reason not to clean up.